app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
That is not correct. I merged master into this branch after #60 was merged. Have you checked out the latest version of this branch to your machine?

https://github.com/ywwg/ffagc/pull/61/commits/023b4db97fee4bb19afba309a247d12f6aaaf115